test/Microsoft.AspNet.Mvc.FunctionalTests/DefaultValuesTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.Framework.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class DefaultValuesTest
```

Comment:
I tried also writing a test for `ViewComponents` to verify that they **don't** support `[DefaultValue(...)]` but it was impossible. VCs use the runtime types of the arguments to dispatch, and you can't call them with a *missing* argument so it's moot.